src/kafka-util/src/bin/kgen.rs

```diff
@@ -548,8 +551,7 @@ async fn main() -> anyhow::Result<()> {
         }
         ValueFormat::Avro => {
             let value_schema = args.avro_value_schema.as_ref().unwrap();
-            let ccsr =
-                ccsr::ClientConfig::new(Url::parse("http://localhost:8081").unwrap()).build();
+            let ccsr = ccsr::ClientConfig::new(args.schema_registry_url).build();
```

Comment:
We should do this for the key schema also

Reply:
#5511 does this for the key schema, it would be great to pull that in!